src/scss/mixins/mixins.scss

```
@@ -0,0 +1,117 @@
+@mixin border-line {
+  border-bottom: $lime-line;
+  border-left: $lime-line;
+}
+
+@mixin font($weight, $size, $height) {
+  font-weight: $weight;
+  font-size: $size;
+  line-height: $height;
+}
+
+@mixin margin-align($top: 0, $right: auto, $bottom: 0, $left: auto) {
+  margin-top: $top;
+  margin-right: $right;
+  margin-bottom: $bottom;
+  margin-left: $left;
+  text-align: center;
+}
+
+@mixin margin-auto {
+  margin: 0 auto;
+}
+
+@mixin width-pic {
+  min-width: 100%;
+  width: 0;
+}
+
+@mixin media-tablet-image {
+  @media (max-width: 1024px) {
+    min-width: 700px;
```

Тут (как и в правиле ниже) можно использовать функцию [min](https://developer.mozilla.org/en-US/docs/Web/CSS/min):
```suggestion
    min-width: min(100%, 700px);
```

тогда можно поубирать все вызовы миксина `width-pic` в медиа-запросах, да и вообще по мне так `width-pic` станет не нужным.